### ISSUE TITLE
colPos in `flux:content.get` does not take CE into account

### ISSUE THREAD

**Issue by pkerling** [FluidTYPO3/flux#1579]
With #1552, `flux:content.get` and `flux:content.render` gained the ability to render not only by specifying the column `name`, but also its `colPos` alternatively. However, the `colPos` does not seem to get interpreted in the context of the CE currently being rendered, which means it is roughly the same as using `v:content.get`. Also, the documentation seems to indicate that this should indeed not be the case, i.e. the `colPos` should be interpreted as I propose. Otherwise there would be little benefit in using the flux view helpers in the first place.

In summary, I think that either the documentation should be changed or the code fixed to actually render the correct column.

**Comment by NamelessCoder** [comment]
Fixed by https://github.com/FluidTYPO3/flux/commit/cc1fb3ca1f1fb1e3c98cf48dab93405a0b5bb018

**Comment by pkerling** [comment]
Thanks!
